wp-content/plugins/core/src/Templates/Content/Panels/Hero.php

```diff
@@ -21,28 +23,97 @@ public function get_data(): array {
 
 	public function get_mapped_panel_data(): array {
 		$data = [
-			'text_color'    => $this->text_color(),
-			'layout'        => $this->get_layout(),
-			'image'         => $this->get_image(),
-			'content_block' => $this->get_content_block(),
+			'row_classes'     => $this->get_row_classes(),
+			'content_classes' => $this->get_content_classes(),
+			'image'           => $this->get_image(),
+			'content_block'   => $this->get_content_block(),
 		];
 
 		return $data;
 	}
 
+	protected function is_split_layout(): bool {
+		if ( $this->panel_vars[ HeroPanel::FIELD_LAYOUT ] === HeroPanel::FIELD_LAYOUT_OPTION_CONTENT_SPLIT_LEFT || $this->panel_vars[ HeroPanel::FIELD_LAYOUT ] === HeroPanel::FIELD_LAYOUT_OPTION_CONTENT_SPLIT_RIGHT ) {
+			return true;
+		}
+
+		return false;
+	}
+
+	protected function get_classes(): string {
+		$classes = [
+			'panel',
+			'site-panel',
```

Comment:
@ryanurban I removed the `s-wrapper` class here, since I didn't want additional padding on the full-bleed background color.  We can revisit with the spacing pass that will come later.

Reply:
I think that makes sense here given this is less of a vertical layout, noted!